examples/observability-to-the-rescue/tracetest/tests/test-with-error.yaml

```diff
@@ -0,0 +1,32 @@
+type: Test
+spec:
+  id: yWoVaiuVR
+  name: Your API call with error
```

Comment:
What is the error in this case? All assertions are waiting `200 OK` status.

Reply:
Maybe it's worth adding a `description` to the test to explain the usecase

Reply:
It makes sense to add a description. This is a test that is expected to fail, to simulate the problem that I'm presenting on the talk.
Context: https://www.loom.com/share/ab3579b2f5fa4fd4af58e088bcad8513?t=852&sid=2b4e5820-2e11-4e42-a9ae-08f33a594e36 (at the 14 min of the talk)